src/main/java/com/java_professional/cherednichenko/homework/lesson_7/Main.java

```diff
@@ -0,0 +1,31 @@
+package com.java_professional.cherednichenko.homework.lesson_7;
+
+import com.java_professional.tsarenko.classwork.lesson7.ex_001_Annotations.methods.AnotherDog;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+
```

Comment:
Add description of this task
```suggestion
/**
*  Задание 1
*  Написать калькулятор с использованием аннотаций, аннотация передает 2 параметра. 
*  Сделать полную документацию всех полей, методов, конструкторов класса используя аннотацию
*  Documented, либо же документационные комментарии.
 */
```